### DISCUSSION TITLE
Docstring of `jax.scipy.interpolate.RegularGridInterpolator` does not mention extrapolation behavior

### DISCUSSION THREAD

**Discussion by timmens** [jax-ml/jax#22031] [General] [answered]
### Problem Description

In the documentation of scipy's [`scipy.interpolate.RegularGridInterpolator`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.RegularGridInterpolator.html) you can find a sentence on extrapolation in the description of the `fill_value` argument:
> #### fill_value: *float or None, optional*
> The value to use for points outside of the interpolation domain. **If None, values outside the domain are extrapolated.**
> Default is `np.nan`.

The documentation of [`jax.scipy.interpolate.RegularGridInterpolator`](https://jax.readthedocs.io/en/latest/_autosummary/jax.scipy.interpolate.RegularGridInterpolator.html#jax.scipy.interpolate.RegularGridInterpolator) just states

> #### Parameters:
> - ...
> - **fill_value** – value returned for points outside the grid, defaults to NaN.

There is no mention of any extrapolation behavior in JAX's source code either. However, passing `fill_value=None` to JAX's `RegularGridInterpolator` results in an extrapolation. (In my tests, it also produces the same results as the scipy function.)

### Request

I think this behavior should be listed in the JAX function's docstring. Since this problem concerns only a docstring, I did not want to open an issue, but I can do that or open a PR with a fix if necessary.

**Comment by jakevdp** [comment, marked as answer]
Seems reasonable, thanks! Would you like to put together a PR with the change?

**Reply by timmens** [reply]
Yes, I can do that!
